Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM openjdk:11.0.10-jdk-slim
 
 RUN mkdir /javavulny /app
 COPY . /javavulny/
+RUN sed -i 's/localhost\:5432/db\:5432/' /javavulny/src/main/resources/application-postgresql.properties
```

Comment:
Should we do this inline replacement, or should we add a specific docker profile and set the correct env var to enable these property changes when we're running in docker?

I'm happy to help do this the Spring way - just LMK 😄

Reply:
I think this is OK, just want to make sure it works both ways and isn't overly complicated.